opal/core/pathway/static/js/pathway/controllers/find_patient.js

```diff
@@ -25,10 +25,11 @@ angular.module('opal.controllers').controller('FindPatientCtrl',
 
     scope.new_patient = function(result){
         scope.state = 'editing_demographics';
+        scope.editing.demographics = [{}];
```

Comment:
I don't think I follow why we need this as well as `scope.demographics` ? (Or indeed at all?)